File: app/src/main/java/com/intelegain/agora/model/News.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by suraj.m on 6/9/17.
 */

public class News {
    @SerializedName("NewsTitle")
    @Expose
    private String newsTitle;
    @SerializedName("NewsDate")
    @Expose
    private String newsDate;

    public String getNewsTitle() {
        return newsTitle;
    }

    public void setNewsTitle(String newsTitle) {
        this.newsTitle = newsTitle;
    }

    public String getNewsDate() {
        return newsDate;
    }

    public void setNewsDate(String newsDate) {
        this.newsDate = newsDate;
    }
}

File: app/src/main/java/com/intelegain/agora/model/News.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by suraj.m on 6/9/17.
 */
class News {
    @SerializedName("NewsTitle")
    @Expose
    var newsTitle: String? = null
    @SerializedName("NewsDate")
    @Expose
    var newsDate: String? = null

}